module/module.go

```diff
@@ -228,10 +229,10 @@ func NewModule(ctx context.Context, address string, logger logging.Logger) (*Mod
 		return nil, err
 	}
 
-	// attempt to construct a PeerConnection
-	pc, err := webrtc.NewPeerConnection(webrtc.Configuration{})
+	pc, err := rgrpc.NewLocalPeerConnection(golog.Global())
```

Comment:
This is a non breaking change that makes modules only accept peer conenctions from viam servers that come from ipv4 loopback addresses ex: 127.0.0.1